server/src/controllers/maproom/v2/cells/enemyCell.ts

```diff
@@ -0,0 +1,39 @@
+import { WorldMapCell } from "../../../../models/worldmapcell.model";
```

Comment:
Is there a good reason why we created this file? Unless it is for testing purposes? I'm not sure there is such thing as an 'enemy' cell? It would just be other user's homebase, outposts and wild monster yards.

I could be wrong about my thoughts here but:

Instead of defining different cell types e.g. `enemyCell`, `wildmonsterCell`, `outpostCell` maybe we should just have a single blueprint for a cell? And then switch out the 'type', etc. Let me know what you think.

Reply:
Yes, it is just for testing. and yes we should use only one